Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM harbor.intgdc.com/tools/gdc-java-8-jre:d0f5e08
 
 ARG GIT_COMMIT=unspecified
```

Comment:
move args and labels at the end of the dockerfile

Reply:
done